javascript-run-nested-promises-in-parallel/README.md

```diff
@@ -0,0 +1,9 @@
+# Run nested promises in parallel
+
```

Comment:
Please add a description that we want to achieve maximum parallelism and how that is visible in the logs.

Reply:
Please take a look at the changes. That's the best wordings I could come up, proposals welcome.